lightwood/data/encoded_ds.py

```diff
@@ -81,13 +81,25 @@ def get_encoded_column_data(self, column_name: str) -> torch.Tensor:
             raise Exception(f'The encoder: {self.encoders[column_name]} for column: {column_name} does not return a Tensor !')
         return encoded_data
 
+    def get_encoded_data(self, include_target=True) -> torch.Tensor:
+        encoded_dfs = []
+        for col in self.data_frame.columns:
+            if (include_target or col != self.target) and self.encoders.get(col, False):
+                encoded_dfs.append(self.get_encoded_column_data(col))
+
+        return torch.cat(encoded_dfs, 1)
+
+    def clear_cache(self):
```

Comment:
This doesn't seem to be called anywhere. If so I'd eer on the side of removing it from this and contacted encoded ds.

Or does this lead to e.g. extra pickling volume during `save` ? (If so it should be called there)

Reply:
It's called in the column importance bit. Given it's just one call, I could replicate the behavior as a procedure in the analysis rather than as methods in the classes, but I'd keep it as it is.